packages/app-desktop/gui/NoteList/NoteList.tsx

```diff
@@ -425,6 +426,17 @@ class NoteListComponent extends React.Component {
 		}
 	}
 
+	fitItemHeight() {
+		// If a note item are styled by CSS, its height is considered.
+		// NOTE: this.itemHeight doesn't need to be in this.state.
+		const ref = Object.values<any>(this.itemAnchorRefs_)[0];
+		const height = ref && ref.current && ref.current.getHeight();
+		if (height > 0) {
```

Comment:
If the height cannot be determined, shouldn't we set a default (like 34, which is the current item height default) ?

Reply:
Yes. If height is undefined or null, itemHeight is not touched and its default value (34) remains.